website/snippets/connect-starburst-trino/schema-db-fields.md

```diff
@@ -1,3 +1,3 @@
 [comment: For context, the section title used for this snippet is "Schemas and databases" ]: # 
 
-When selecting the database (catalog) and the schema, make sure the user has read and write access to both the provided database (catalog) and schema. This selection does not limit your ability to query the catalog. Instead, they serve as the default location for where tables and views are materialized. This _default_ can be changed later from within your dbt project.
\ No newline at end of file
+When selecting the catalog and the schema, make sure the user has read and write access to both. This selection does not limit your ability to query the catalog. Instead, they serve as the default location for where tables and views are materialized. In addition, the Trino connector used in the catalog must support creating tables and views in the catalog. This _default_ can be changed later from within your dbt project.
```

Comment:
@mosabua can you tell me more about your addition? do some Trino connectors not support creating tables and views?
> Trino connector used in the catalog must support creating tables and views in the catalog

 Or, do you mean that the connector must be correctly configured so that the cluster can make tables and views?

Reply:
There are connectors that only support reading data, others support table creation, but not view creation, and so on. 

I am not sure what the hard requirement for a catalog is and I also cant conclude from the comment from @hovaesco 

At the moment I think table creation is a hard requirement so I will update the docs accordingly.

Reply:
> Also ... if this just sets the default for table and view creation.. can your SQL statements in dbt also access other catalogs?

Yes, the connection profile catalog only defines to which catalog data will be loaded. For reading the data the only requirement is just to set up a catalog correctly.

> At the moment I think table creation is a hard requirement so I will update the docs accordingly.

True, that's the minimum, since I believe all connectors which support creating view support creating tables.

Reply:
Great.. I hope that makes sense @dataders